### PR TITLE
Update google_sheets.markdown

### PR DESCRIPTION
**Pull request by danielbrunt57**
Provide an example service call to append data to a Google sheet.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/google_sheets.markdown [hunk 1]
```diff
@@ -83,4 +83,21 @@ You can use the service `google_sheets.append_sheet` to add a row of data to the
...
+  worksheet: Clarity Charging
+  data:
+    Date: "{{ now().strftime('%-d-%b-%y') }}"
+    Start: "{{ states('input_number.car_charging_start')|float(0) }}"
+    End: "{{ states('input_number.car_charging_end')|float(0) }}"
+    Rate: "{{ states('input_number.bchydro_current_rate')|float(0) }}"
+    Cost: >-
+      {{ (states('input_number.car_charging_end')|float(0) -
+      states('input_number.car_charging_start')|float(0) ) *
+      states('input_number.bchydro_current_rate')|float(0) }}
```
Comment (frenck) [inline source/_integrations/google_sheets.markdown]: Let's simplify the example to make it more clear.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/google_sheets.markdown (hunk 1 above)
Comment (danielbrunt57) [reply source/_integrations/google_sheets.markdown]: How simple do you think it should be?

Comment (frenck) [reply source/_integrations/google_sheets.markdown]: Well, we could use a smaller template, as in, we don't need to make it so specific and have it using 3 entities (for example).

Comment (danielbrunt57) [reply source/_integrations/google_sheets.markdown]: How about:
```
data:
  config_entry: 1b4a46c6cba0677bbfb5a8c53e8618b0
  worksheet: Car Charging
  data:
    Date: "{{ now().strftime('%-d-%b-%y') }}"
    KWh: "{{ states('input_number.car_charging_kwh')|float(0) }}"
    Cost: "{{ states('input_number.car_charging_cost')|float(0) }}"
```

Comment (frenck) [reply source/_integrations/google_sheets.markdown]: Yeah sure 👍 That is much more simple

Comment (danielbrunt57) [reply source/_integrations/google_sheets.markdown]: Changed.

**Review by danielbrunt57 — COMMENTED** [review]
Simplified the data: example.

File: source/_integrations/google_sheets.markdown [hunk 2]
```diff
@@ -83,4 +83,18 @@ You can use the service `google_sheets.append_sheet` to add a row of data to the
 | `worksheet` | yes | Name of the worksheet. Defaults to the first one in the document. | Sheet1
 | `data` | no | Data to be appended to the worksheet. This puts the data on a new row, one value per column. | {"hello": world, "cool": True, "count": 5}
 
+```yaml
+# Example service call
+service: google_sheets.append_sheet
+data:
+  config_entry: 1b4a46c6cba0677bbfb5a8c53e8618b0
+  worksheet: Clarity Charging
+data:
```
Comment (frenck) [inline source/_integrations/google_sheets.markdown]: data is an duplicate object key in this example

Comment (danielbrunt57) [reply source/_integrations/google_sheets.markdown]: My bad!  Copy/paste error...

**Review by danielbrunt57 — COMMENTED** [review]
Edited service call parameters

File: source/_integrations/google_sheets.markdown [hunk 3]
```diff
@@ -83,4 +83,15 @@ You can use the service `google_sheets.append_sheet` to add a row of data to the
 | `worksheet` | yes | Name of the worksheet. Defaults to the first one in the document. | Sheet1
 | `data` | no | Data to be appended to the worksheet. This puts the data on a new row, one value per column. | {"hello": world, "cool": True, "count": 5}
 
+```yaml
+# Example service call
+service: google_sheets.append_sheet
+data:
+  config_entry: 1b4a46c6cba0677bbfb5a8c53e8618b0
+  worksheet: Car Charging
```
Comment (frenck) [inline source/_integrations/google_sheets.markdown]:
```suggestion
  worksheet: "Car Charging"
```

File: source/_integrations/google_sheets.markdown [hunk 4]
```diff
@@ -83,4 +83,15 @@ You can use the service `google_sheets.append_sheet` to add a row of data to the
 | `worksheet` | yes | Name of the worksheet. Defaults to the first one in the document. | Sheet1
 | `data` | no | Data to be appended to the worksheet. This puts the data on a new row, one value per column. | {"hello": world, "cool": True, "count": 5}
 
+```yaml
```
Comment (frenck) [inline source/_integrations/google_sheets.markdown:88]:
````suggestion
{% raw %}

```yaml
````

File: source/_integrations/google_sheets.markdown [hunk 5]
```diff
@@ -83,4 +83,15 @@ You can use the service `google_sheets.append_sheet` to add a row of data to the
...
+# Example service call
+service: google_sheets.append_sheet
+data:
+  config_entry: 1b4a46c6cba0677bbfb5a8c53e8618b0
+  worksheet: Car Charging
+  data:
+    Date: "{{ now().strftime('%-d-%b-%y') }}"
+    KWh: "{{ states('input_number.car_charging_kwh')|float(0) }}"
+    Cost: "{{ states('input_number.car_charging_cost')|float(0) }}"
+ ```
```
Comment (frenck) [inline source/_integrations/google_sheets.markdown]:
````suggestion
```

{% raw %}

````

File: source/_integrations/google_sheets.markdown [hunk 6]
```diff
@@ -83,4 +83,20 @@ You can use the service `google_sheets.append_sheet` to add a row of data to the
...
+data:
+  config_entry: 1b4a46c6cba0677bbfb5a8c53e8618b0
+  worksheet: "Car Charging"
+  data:
+    Date: "{{ now().strftime('%-d-%b-%y') }}"
+    KWh: "{{ states('input_number.car_charging_kwh')|float(0) }}"
+    Cost: "{{ states('input_number.car_charging_cost')|float(0) }}"
+```
+
+{% raw %}
```
Comment (frenck) [inline source/_integrations/google_sheets.markdown]:
```suggestion
{% endraw %}
```

**Review by frenck — APPROVED** [review]
Thanks, @danielbrunt57 👍

**Pull request merged**